forgot_password/__init__.py

```diff
@@ -194,3 +193,13 @@ def reset_password_handler(request):
             return skygear.Response(body, content_type='text/html')
 
     return reset_password_response(**template_params)
+
+
+def init():
```

Comment:
While this works, I am not in favour of this.

I have written another plugin that wraps handler function in a function so that registration only occur after some conditions:

```
def create_plugin():
    @skygear.handler('handler-name')
    def handler_function(request):
        pass
```

This is also a hack, but we don’t suddenly use decorators as functions.